.github/workflows/sync-redirects.yaml

```diff
@@ -0,0 +1,30 @@
+name: Sync RTD redirects
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - docs/redirects.yml
+      - .github/workflows/sync-redirects.yaml
+
+  # Manually triggered using GitHub's UI
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+
+      - name: Upgrade Python toolchain
+        run: python3 -m pip install --upgrade pip setuptools wheel
+
+      - name: Install readthedocs-cli
+        run: python3 -m pip install readthedocs-cli
+
+      - name: Sync redirects
+        run: rtd projects nextstrain-ncov redirects sync -f doc/redirects.yaml --wet-run
```

Comment:
```suggestion
        run: rtd projects nextstrain-ncov redirects sync -f docs/redirects.yaml --wet-run
```

Reply:
Fixed in [force-push](https://github.com/nextstrain/ncov/compare/7b9e45d957c3522b821c5134d76ded0fa175d743..0789007d227d4b9fec63902fd359d8e80cbb6d08). Also fixed an incorrect filename.